queries.sql

```diff
@@ -1,3 +1,18 @@
 /*Queries that provide answers to the questions from all projects.*/
+-- SELECT * from animals WHERE name = 'Luna';
```

Comment:
- [ ] Let's remove commented code. Leaving commented code in your codebase is a bad practice.

Reply:
Done